### PR TITLE
Add HUD button that displays your SSS role and allies

### LINKED ISSUE #1702: SSS role should be more prominently displayed to the player
It's easy for it to get lost in chat. Maybe some kind of role indicator should be included in the HUD?

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Fixes #1702

### CODE REVIEW

File: Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            if (announceTo.Count == 0)
+            {
+                return;
+            }
+
+            var antagonistIds = _antagonists.Select(t => t.Owner.Name).ToHashSet();
+            var message = new SuspicionAlliesMessage(antagonistIds);
+
+            foreach (var channel in announceTo)
+            foreach (var traitor in _antagonists)
```
Comment (RemieRichards) [inline Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs:68]: Disgusting.

Comment (RemieRichards) [reply Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs:68]: Oh god you can actually stack syntax like this infinitely...

Comment (DrSmugleaf) [reply Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs:68]: But of course:

image

Comment (RemieRichards) [reply Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs:68]: is that from mafia smug...

Comment (RemieRichards) [reply Content.Server/GameObjects/EntitySystems/SuspicionRoleSystem.cs:68]: it is!!!!

**Pull request merged**